apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -1097,6 +1097,10 @@
         "firmwareUpdate": {
           "inactive": "Software check",
           "active": "Checking for software updates",
+          "updateAvailable": "Software update available 2.2.0",
```

Comment:
Why is this hardcoded?
I'm guessing the version needs to come from an API call, not the language files?

Reply:
This PR only add UI without logic. So everything is hardcoded for now. We will add the logic in an other PR (And remove all hardcoded data)